### ISSUE TITLE
Does not work anymore?

### ISSUE THREAD

**Issue by Evilbot83** [SnosMe/awakened-poe-trade#908]
Starting the new league Awakened does not work for me anymore. 

Произошла ошибка при парсинге предмета
Скорее всего, это ошибка, и вы можете сообщить о ней на GitHub.

**Comment by sephira1990** [comment]
<img width="338" alt="Awakened_PoE_Trade_UmtuMOY1jp" src="https://user-images.githubusercontent.com/109391215/230722982-4974d871-8e8f-460f-ac6b-d2a43c7a7b33.png">
<img width="453" alt="NVIDIA_Share_nrS7PhlpYe" src="https://user-images.githubusercontent.com/109391215/230722986-8ba4965f-d0a2-450e-b8a4-79e31179bbbc.png">

**Comment by SnosMe** [comment]
Just on click on Browser as it says, wait a bit till it checks the browser, and then you can Retry

**Comment by InqClept** [comment]
> Just click on Browser button as it says, wait a bit till it checks the browser, and then you can Retry

![image](https://user-images.githubusercontent.com/130231593/230725291-25a7e46f-4264-47fd-b558-2d12538bed61.png)

i click in button....

**Comment by SnosMe** [comment]
yes, this is what you see after it passes security check. Now click Retryt 🤷

**Comment by InqClept** [comment]
> yes, this is what you see after it passes security check. Now click Retry 🤷

yep. ty

**Comment by Evilbot83** [comment]
> Just click on Browser button as it says, wait a bit till it checks the browser, and then you can Retry

it does not offer me to open up the browser

**Comment by SnosMe** [comment]
So give a screenshot of what you got

**Comment by DelsideaD** [comment]
![Screenshot_26](https://user-images.githubusercontent.com/130236150/230729164-7d5ae2fa-54f4-4c7c-a569-426c9d24c65c.png)
Same thing, can't load leagues. When I click the Retry button, nothing happens.

**Comment by SnosMe** [comment]
![image](https://user-images.githubusercontent.com/4292308/230729351-58c59385-a7c5-4293-b96d-de5ccfe5ff33.png)

Follow instructions word by word, you're currently in the settings!

**Comment by DelsideaD** [comment]
> Follow instructions word by word, you're currently in the settings!

I'm sorry, everything worked out, thanks

**Comment by sephira1990** [comment]
<img width="329" alt="NVIDIA_Share_g0Ju3w5ZqO" src="https://user-images.githubusercontent.com/109391215/230731324-406a8e7b-0cc2-4286-bfce-bc5e61cb7b61.png">

**Comment by Evilbot83** [comment]
> <img alt="NVIDIA_Share_g0Ju3w5ZqO" width="329" src="https://user-images.githubusercontent.com/109391215/230731324-406a8e7b-0cc2-4286-bfce-bc5e61cb7b61.png">

I almost done it. But now got same error
